### DISCUSSION TITLE
Top bar icons (+, X min max) doesn't work - Windows version

### DISCUSSION THREAD

**Discussion by csfalcao** [thanhlmm/refi-app#46] [General]
The close icon (X) doesn't work, as long other in the top bar. It will only minimize when double click in any place in the grey bar, and vice versa. The + sign doesn't work too.

**Comment by csfalcao** [comment]
Just Windows

**Reply by thanhlmm** [reply]
Thanks. I will check it as soon as I get Window env 😅
